resolvers/game.go

```diff
@@ -36,18 +36,26 @@ func (r *Resolvers) CreateGame(p graphql.ResolveParams) (interface{}, error) {
 		return nil, err
 	}
 
+	s := p.Args["size"]
+	var size int
+	if s == nil {
```

Comment:
do you think this should be optional? should a client just always specify a board size?

Reply:
I was debating it with myself when I was doing it. It will make a breaking change with the client but it's a super quick fix

Reply:
I don't think we should worry too much about breaking changes at this point. w/ semantic versioning, any minor version change <1.0 can be considered breaking. But honestly until we reach MVP I don't think we even need to worry about versioning too much.

Reply:
Yeah, put pressure on me to fix the client. At this stage it's OK 👍